data/autoyast_kvm/sles11sp4x32_PRG.xml

```diff
@@ -721,6 +721,7 @@
       <package>yast2-schema</package>
       <package>yast2-trans-en_US</package>
       <package>yudit</package>
+      <package>qemu-guest-agent</package>
```

Comment:
What will we do with this package?

Reply:
imho, this package (e.g qemu-guest-agent) , always help us get guest IP address. 
For example:
```
# virsh qemu-agent-command sles-15-sp3-64-fv-def-net '{"execute":"guest-network-get-interfaces"}' --pretty | grep 192.168
          "ip-address": "192.168.123.10",   
```

Reply:
Exactly, that was a request from @srlemke to include it. I think he still plays with it but the reason was to get the guest IP address.

Reply:
And is this `qemu-guest-agent` package now included on every guest? I remember it was blocking the automatic installation for some versions or?

Reply:
Yes it is included on every guest and it also works on every guest. The culprit for the issue that you remember was a race condition in `waitfor_guests.pm`, I wrongly accuest `qemu-guest-agent` for it back then.